projects/js-themes-toolkit/scripts/updateThemeDeps.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * SPDX-FileCopyrightText: © 2017 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+/* eslint-disable no-console, @liferay/no-dynamic-require */
+
+const fs = require('fs');
+const https = require('https');
+
+const DEV_DEPENDENCIES_PATH = require.resolve(
+	'../packages/liferay-theme-tasks/lib/devDependencies.js'
+);
+const SIGNIFICANT_DEPS = [
+	'liferay-frontend-css-common',
+	'liferay-frontend-theme-styled',
+	'liferay-frontend-theme-unstyled',
+];
+
+async function main(argv) {
+	const {theme: themeDeps} = require(DEV_DEPENDENCIES_PATH);
+
+	const currentVersions = {};
+	const wantedVersions = {};
+
+	for (const dxpVersion of Object.keys(themeDeps)) {
+		const {default: depVersions} = themeDeps[dxpVersion];
+
+		console.log('');
+		console.log('');
+		console.log(`== DXP ${dxpVersion} ========`);
+
+		currentVersions[dxpVersion] = {};
+		wantedVersions[dxpVersion] = {};
+
+		for (const dep of SIGNIFICANT_DEPS) {
+			const depVersion = depVersions[dep];
+
+			if (!depVersion) {
+				continue;
+			}
+
+			const versionSeries = depVersion.replace(/\..*/, '');
+
+			console.log('');
+			console.log(dep);
+			console.log('    Current version:', depVersion);
+			console.log('    Versions series:', versionSeries);
+			process.stdout.write('    Latest version:  ');
+
+			const npmVersions = await getNpmVersions(dep);
+
+			const latestVersion = getLatestVersion(npmVersions, versionSeries);
+
+			console.log(latestVersion);
+
+			currentVersions[dxpVersion][dep] = depVersion;
+			wantedVersions[dxpVersion][dep] = latestVersion;
+		}
+	}
+
+	console.log('');
+	console.log('');
+
+	if (argv[2] === '--check' || argv[2] === '-c') {
```

Comment:
Not a big deal, but you could also do `argv.includes('--check')` in case at a later date we add another arg that may be in a different order.

Reply:
True :+1: